Bottom: 31d7ebb1846fb13e6388fe131b598171d473d00a
Top:    163f73e69bbc6422f86a81d86bef97a56c493c23
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-06-29 15:36:11 +0800

Refresh of net-introduce-skb_get_swhash

---

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index f769543..bea166d 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -930,7 +930,7 @@ static inline __u32 skb_get_hash(struct sk_buff *skb)
 {
 	if (!skb->l4_hash)
 		return skb_get_sw_hash(skb);
-        else if (skb->sw_hash)
+	else if (skb->sw_hash)
 		return skb->sw_hash;
 
 	return skb->hash;
